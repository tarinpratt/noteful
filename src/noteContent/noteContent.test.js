import React from 'react';
import ReactDOM from 'react-dom';
import { BrowserRouter } from 'react-router-dom';
import NoteContent from './noteContent'


it('renders without crashing', () => {
  const div = document.createElement('div')
  ReactDOM.render(
    <BrowserRouter>
    <NoteContent />
    </BrowserRouter>, 
  div
  )
  ReactDOM.unmountComponentAtNode(div);
})

