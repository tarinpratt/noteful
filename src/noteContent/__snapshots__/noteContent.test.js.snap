// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoteContent component renders Note Content 1`] = `
<div
  className="selectedNote"
>
  <Note
    onDeleteNote={[Function]}
  />
  <div
    className="noteContent"
  >
    <p
      key="0"
    />
  </div>
</div>
`;

exports[`NoteContent component renders a Note with note prop 1`] = `
<Note
  onDeleteNote={[Function]}
/>
`;
